README.rst

```diff
@@ -37,6 +37,19 @@ Alternatively to get more up-to-date features:
 
 check the changelog_ for information about features included on `master` but not yet released.
 
+Dev Env Setup
+-------------
+
+Installation:
+    Current Python version supported for development is :code:`v3.5.6`.
```

Comment:
nit: Let's say refer to `setup.py`, `python_requires` line to get the minimum supported Python version. That way we don't have to remember to update two places